### DISCUSSION TITLE
Importing BMECat data (XML) into Pimcore

### DISCUSSION THREAD

**Discussion by wumi20** [pimcore/pimcore#8125] [Q&A / Support] [answered]
Hi guys

Are there any converters to import BMEcat data into Pimcore, or any solutions to migrate BMEcat data into the Pimcore directly?

Thanks

**Comment by aarongerig** [comment, marked as answer]
Hi @wumi20 

Have a look at [DataDefinitions](https://github.com/w-vision/DataDefinitions). You can create your own import provider (BMEcat), which would be quite similar to the [XML provider](https://github.com/w-vision/DataDefinitions/blob/master/src/DataDefinitionsBundle/Provider/XmlProvider.php).

Hope that helps!
